api/app/tests/utils/test_fire_centres.json

```diff
@@ -0,0 +1,2737 @@
+{
```

Comment:
Please delete, already exists in: `api/app/tests/hfi/test_fire_centres.json` on main

Reply:
Marked resolved but doesn't appear to have been deleted...? Same with the 2 other test JSONs

Reply:
@JulianForeman , please address before resolving